# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/moz.build
# Commit: 48ce2b670f32
# Full Hash: 48ce2b670f32691b0537c13c165ec8459d74907c
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2019-09-25 09:36:42
# Regressor Bug: 1577643
# File Overlap Count: 1
# Description:
#   Bug 1577643 - Implement a security info class for the quic transport. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44073
# ==============================================================================

diff -r 2a39731bf265 -r 48ce2b670f32 netwerk/protocol/http/moz.build
--- a/netwerk/protocol/http/moz.build	Tue Sep 24 22:40:20 2019 +0000
+++ b/netwerk/protocol/http/moz.build	Tue Sep 24 20:56:39 2019 +0000
@@ -108,6 +108,7 @@
     'NullHttpChannel.cpp',
     'NullHttpTransaction.cpp',
     'ParentChannelListener.cpp',
+    'QuicSocketControl.cpp',
     'TunnelUtils.cpp',
 ]
 